umake/__init__.py

```diff
@@ -124,8 +138,10 @@ def main():
 
     mainloop = MainLoop()
 
-    # load frameworks and initialize parser
-    load_frameworks()
+    # load frameworks
+    set_loading_frameworks(sys.argv)
```

Comment:
hum, I guess you are now loading frameworks twice, as it's called as well later on (loading_frameworks()).

Please ensure this returns a boolean, and load frameworks only once, but do it in the correct place ;)